### ISSUE TITLE
Cannot read property 'name' of undefined when installing addons

### ISSUE THREAD

**Issue by BenjaminHorn** [ember-cli/ember-cli#4102]
I have a new project, but I cant install addons:

![ember-cli](https://cloud.githubusercontent.com/assets/5570232/7683632/b56d5212-fd80-11e4-9465-7b20033c4b77.png)

I also tried with different addons, e.x.: ember-cli-flash but same error message.
node: 0.12.3
npm: 2.5.1.
```
  "dependencies": {
    "ember": "1.12.0",
    "ember-cli-shims": "ember-cli/ember-cli-shims#0.0.3",
    "ember-cli-test-loader": "ember-cli-test-loader#0.1.3",
    "ember-data": "1.0.0-beta.17",
    "ember-load-initializers": "ember-cli/ember-load-initializers#0.1.4",
    "ember-qunit": "0.3.3",
    "ember-qunit-notifications": "0.0.7",
    "ember-resolver": "~0.1.15",
    "jquery": "^1.11.1",
    "loader.js": "ember-cli/loader.js#3.2.0",
    "qunit": "~1.17.1"
  }
```

```
  "devDependencies": {
    "broccoli-asset-rev": "^2.0.2",
    "ember-cli": "0.2.5",
    "ember-cli-app-version": "0.3.3",
    "ember-cli-babel": "^5.0.0",
    "ember-cli-content-security-policy": "0.4.0",
    "ember-cli-dependency-checker": "^1.0.0",
    "ember-cli-flash": "1.1.4",
    "ember-cli-htmlbars": "0.7.6",
    "ember-cli-ic-ajax": "0.1.1",
    "ember-cli-inject-live-reload": "^1.3.0",
    "ember-cli-qunit": "0.3.13",
    "ember-cli-uglify": "^1.0.1",
    "ember-data": "1.0.0-beta.17",
    "ember-disable-proxy-controllers": "^0.7.0",
    "ember-export-application-global": "^1.0.2",
    "ember-select-2": "1.3.0"
  }
```

**Comment by abuiles** [comment]
can you please do `npm install ember-cli-dependency-checker@latest --save-dev` the issue is coming from there.

**Comment by BenjaminHorn** [comment]
Thanks for the fast help. I did it like u suggested. I had *npm WARN package.json ember-disable-proxy-controllers@0.7.0 No repository field.*

but it solved the problem.

**Comment by abuiles** [comment]
@cibernox ^^

**Comment by cibernox** [comment]
Humm, what does that error message mean @abuiles ? My addon does not have a blueprint (it doesn't need one).

**Comment by rwjblue** [comment]
@cibernox - You fixed the issue in https://github.com/cibernox/ember-disable-proxy-controllers/commit/b43d8c5c942f8d9c61336ffd70d41b348620a732, but have not released.

**Comment by cibernox** [comment]
@rwjblue I didn't know that those fields were important. Published 1.0.0
